hydra-node/src/Hydra/Chain/Blockfrost.hs

```diff
@@ -230,12 +242,46 @@ blockfrostChainFollow tracer prj chainPoint handler wallet = do
   retryPolicy :: Double -> RetryPolicyM m
   retryPolicy blockTime' = constantDelay (truncate blockTime' * 1000 * 1000)
 
-  loop stateTVar = do
+  catchUpToLatest currentHash stateTVar = do
+    latestBlock <- Blockfrost.runBlockfrostM prj BlockfrostAPI.getLatestBlock
+    let targetHash = BlockfrostAPI._blockHash latestBlock
+    let targetHeight = BlockfrostAPI._blockHeight latestBlock
```

Comment:
When I checked out the code I noticed you are not using `targetHeight` and `currentHeight` below? Is the code in this PR complete or this is just not cleaned up and we don't allow warnings in our code  base. Perhaps you wanted to use  the block height but forgot about it? @awcjack